.github/ISSUE_TEMPLATE/generic-feature-template.md

```diff
@@ -7,19 +7,65 @@ assignees: ''
 
 ---
 
-As a: [developer/user/instrument/project requirement]
+# Feature Description
 
-### Acceptance Criteria
-_What is the acceptance criteria?_
-- [ ] a
-- [ ] b
+**Aim to not delete parts of the template.**
 
-### Extra Information
-_Why it is needed?_
+<sub>Fill in the blanks. _Example: "As a user, I would like to reset my password so that I can regain access to my account if I forget my password."_</sub>
 
-_Where required files/links are_
+[comment]: <> (FILL IN BELOW)
+~~~
+> As a [developer/user/instrument/project requirement], I would like to [feature or functionality] so that [reason/benefit].
+~~~
```

Comment:
I wonder if we should take these out of backticks - may be easier to edit as you don't have to remove the backticks first.

Reply:
Okay- I've just swapped these out. Forgot that tildes are usually used to create a codeblock